src/App.js

```diff
@@ -2,13 +2,39 @@ import React from "react";
 import "./App.css";
 
 class App extends React.Component {
+  constructor() {
+    super()
+    this.state = {
+      score: 0,
+      clicky: 1
+    }
+  }
+  onClicking = () => {
+    this.setState({ score: this.state.score + this.state.clicky })
+  }
+  prompt = () => {
+    alert("You can't afford that!")
+  }
+  currentScore = () => {
+    this.state.clicky = this.state.clicky + 1
+    this.setState({ score: this.state.score - 10 })
+  }
```

Comment:
```suggestion
  currentScore = () => {
  const { score, clicky } = this.state
    this.setState({ score: score - 10, clicky: clicky + 1 })
  }
```
again, you're modifying state directly here on line 19. Modifying state directly is bad practice, creates bugs, and goes against "The React Way". **Always use setState, don't modify state directly!**